src/openrct2-ui/windows/EditorObjectiveOptions.cpp

```diff
@@ -263,7 +263,6 @@ rct_window* window_editor_objective_options_open()
     w->pressed_widgets = 0;
     w->hold_down_widgets = window_editor_objective_options_page_hold_down_widgets[WINDOW_EDITOR_OBJECTIVE_OPTIONS_PAGE_MAIN];
     window_init_scroll_widgets(w);
-    w->var_4AE = 0;
```

Comment:
You removed this assignment, but didn't remove `var_4AE` from the `rct_window` struct, was that intentional?

Reply:
Yes, this is intentional. It is not used in this window and initialised to 0 in `window_create()` anyway. The reason I didn't remove it entirely is that it's still used in the Player window.